spec/models/wishlist_item_spec.rb

```diff
@@ -48,4 +48,14 @@
         .to match_array([item_2.id, item_1.id, item_3.id])
     end
   end
+  
+  describe 'When a WishlistItem with pledges is deleted' do
+    it 'Destroys the pledges too' do
+    wishlist_item = FactoryGirl.create(:wishlist_item)
+    pledge        = FactoryGirl.create(:pledge)
```

Comment:
They actually changed the gem name yesterday–it's now `FactoryBot`. Dropping the class prefix will fix this.